exec/java-exec/src/test/java/org/apache/drill/exec/testing/TestPauseInjection.java

```diff
@@ -150,66 +150,61 @@ public void pauseOnSpecificBit() {
-    final DrillConfig drillConfig = zkHelper.getConfig();
     try {
-      drillbit1 = Drillbit.start(drillConfig, remoteServiceSet);
-      drillbit2 = Drillbit.start(drillConfig, remoteServiceSet);
-    } catch (final DrillbitStartupException e) {
-      throw new RuntimeException("Failed to start two drillbits.", e);
-    }
-
-    final DrillbitContext drillbitContext1 = drillbit1.getContext();
-    final DrillbitContext drillbitContext2 = drillbit2.getContext();
-
-    final UserSession session = UserSession.Builder.newBuilder()
-      .withCredentials(UserCredentials.newBuilder()
-        .setUserName("foo")
-        .build())
-      .withUserProperties(UserProperties.getDefaultInstance())
-      .withOptionManager(drillbitContext1.getOptionManager())
-      .build();
-
-    final DrillbitEndpoint drillbitEndpoint1 = drillbitContext1.getEndpoint();
-    final String controls = Controls.newBuilder()
-      .addPauseOnBit(DummyClass.class, DummyClass.PAUSES, drillbitEndpoint1)
-      .build();
-
-    ControlsInjectionUtil.setControls(session, controls);
-
-    {
-      final long expectedDuration = 1000L;
-      final ExtendedLatch trigger = new ExtendedLatch(1);
-      final Pointer<Exception> ex = new Pointer<>();
-      final QueryContext queryContext = new QueryContext(session, drillbitContext1, QueryId.getDefaultInstance());
-      (new ResumingThread(queryContext, trigger, ex, expectedDuration)).start();
-
-      // test that the pause happens
-      final DummyClass dummyClass = new DummyClass(queryContext, trigger);
-      final long actualDuration = dummyClass.pauses();
-      assertTrue(String.format("Test should stop for at least %d milliseconds.", expectedDuration),
-        expectedDuration <= actualDuration);
-      assertTrue("No exception should be thrown.", ex.value == null);
+      // Creating two drillbits
+      final Drillbit drillbit1, drillbit2;
+      final DrillConfig drillConfig = zkHelper.getConfig();
       try {
-        queryContext.close();
-      } catch (final Exception e) {
-        fail("Failed to close query context: " + e);
+        drillbit1 = Drillbit.start(drillConfig, remoteServiceSet);
+        drillbit2 = Drillbit.start(drillConfig, remoteServiceSet);
+      } catch (final DrillbitStartupException e) {
+        throw new RuntimeException("Failed to start two drillbits.", e);
       }
-    }
 
-    {
-      final ExtendedLatch trigger = new ExtendedLatch(1);
-      final QueryContext queryContext = new QueryContext(session, drillbitContext2, QueryId.getDefaultInstance());
+      final DrillbitContext drillbitContext1 = drillbit1.getContext();
+      final DrillbitContext drillbitContext2 = drillbit2.getContext();
+
+      final UserSession session = UserSession.Builder.newBuilder().withCredentials(UserCredentials.newBuilder().setUserName("foo").build()).withUserProperties(UserProperties.getDefaultInstance()).withOptionManager(drillbitContext1.getOptionManager()).build();
```

Comment:
Same issue of one big line vs. multi-line.

Reply:
done